frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -1,12 +1,38 @@
-import { connect } from "react-redux";
-import type { State } from "metabase-types/store";
+import { css, Global } from "@emotion/react";
```

Comment:
This combines the previous `GlobalStyles` View and this file. So we don't have them in separate places. We can now use a single file because we can use `useSelector`